deploy/crd/pmem-csi.intel.com_deployments.yaml

```diff
@@ -0,0 +1,183 @@
+
+---
```

Comment:
Where does this file come from? If it is generated, can you add a comment at the top that explains that?

Reply:
The file was generated by `controller-gen` and it does not add any comment. Changed the make target to add this comment after generating the crd file.